.eslintrc.cjs

```diff
@@ -2,6 +2,7 @@ module.exports = {
   extends: [
     "eslint:recommended",
     "plugin:mocha/recommended",
+    "plugin:jest-formatting/recommended",
```

Comment:
why do we need these additinal rules?

Reply:
It was the fastest way to get the rules that ensure proper spacing between test blocks.
I could create custom padding-between-statements rules to capture this, but it would take extra effort.
See their implementation: https://github.com/dangreenisrael/eslint-plugin-jest-formatting/blob/master/src/rules/padding.ts

Reply:
Fair enough, would you mind adding a comment to explain it?

Reply:
done